src/Makefile

```diff
@@ -21,7 +21,7 @@
 PREFIX=/usr/local
 CC=gcc
 CFLAGS=-c -fPIC -Wall -D_REENTRANT $(ADDITIONALFLAGS)
-LIBS=-lc -lorcania
+LIBS=-lc -lorcania $(ADDITIONALLIBS)
```

Comment:
I'm not sure I understand this one, where is this variable defined?

Reply:
It funtions much like the $(ADDITIONALFLAGS) variable for CFLAGS in the line above.  It allows me to pass different linking arguments to the makefile.  In my case I have something like this in my dependency build step:

$(MAKE) -C ulfius/lib/yder/src static-install PREFIX=$(DEPENDENCIES) ADDITIONALFLAGS=-I$(DEPENDENCIES)/include ADDITIONALLIBS=-L$(DEPENDENCIES)/lib

This accounts for the fact that in my build environment I need to link and install libraries to a non-standard location.  This is only necessary when building static libraries, because the system does not keep a cache of static library locations.

Reply:
OK so this allows you to append specific path for librairies. It's not used in the default build, but you can use it in your own build process if you need. Am I correct?

Reply:
If we add this to Yder's Makefile, shouldn't we add it on the other libraries too? (Orcania, Ulfius, Hoel)

Reply:
I only added it to Yder, because that's the only one that needed it for my situation.  It's only necessary for situations where a library is both a dependency of a library and has dependencies on other libraries.  If both of those aren't true, it can be handled by linking options to the package you're building.